### PR TITLE
General fixes

### PR DESCRIPTION
**Pull request by FrancisA2000**
did some fixes in general that help get a better style and mobile first,
no need to specify for what issues I did, because it was mostly style.css.

@Karyum  dont do it.

also added router so we can navigate between pages.

@Karyum dont do it.

### CODE REVIEW

**Comment by Karyum** [comment]
added the `in-progress` label, when youre done with this PR change the label and reassign me

**Comment by Karyum** [comment]
@FrancisA2000 ohh 29 files !! what happened :laughing: , you sure you have the latest master ?

**Comment by Karyum** [comment]
can i also review now or is it still not ready ?

**Comment by FrancisA2000** [comment]
It's ready you can review it

File: public/index.html [hunk 1]
```diff
@@ -3,7 +3,6 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
```
Comment (Karyum) [inline public/index.html]: why have you removed this ?

File: src/components/Chart/index.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  Tooltip,
+  Area,
+  XAxis,
+  YAxis,
+  AreaChart,
+  CartesianGrid,
+  ResponsiveContainer
+} from "recharts";
+import Typography from "@material-ui/core/Typography";
+// import Title from './Title';
```
Comment (Karyum) [inline src/components/Chart/index.jsx]: remove this line if you don't need it

File: src/components/Chart/index.jsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+      <ResponsiveContainer width={800} height={350}>
+      <AreaChart data={data}
+  margin={{ top: 10, right: 30, left: 0, bottom: 0 }}>
+  <defs>
+    <linearGradient id="colorUv" x1="0" y1="0" x2="0" y2="1">
+      <stop offset="5%" stopColor="#8884d8" stopOpacity={0.8}/>
+      <stop offset="95%" stopColor="#8884d8" stopOpacity={0}/>
+    </linearGradient>
+  </defs>
```
Comment (Karyum) [inline src/components/Chart/index.jsx]: this file is a mess please your formatter here

**Review by Karyum — CHANGES_REQUESTED** [review]
before i can merge this PR, you need to sort out your formatter with the rest of the team.

either use eslintrc file or editor config file, because this is becoming a bit of a mess

coordinate with the other teammates on this

File: public/index.html (hunk 1 above)
Comment (FrancisA2000) [reply public/index.html]: It was messing up the profile page, but I will return it

**Review by Karyum — APPROVED** [review]

**Pull request merged**